httpobs/scanner/grader/grade.py

```diff
@@ -318,6 +318,10 @@
         'description': 'Subresource Integrity (SRI) implemented, but external scripts are loaded over http',
         'modifier': -20,
     },
+    'sri-not-implemented-and-same-origin-scripts-not-loaded-securely': {
+        'description': 'Subresource Integrity (SRI) is not implemented, and scripts from a similar origin are not loaded over https',
```

Comment:
This can pass flake8 if you wrap this line. something like

```
    'sri-not-implemented-and-same-origin-scripts-not-loaded-securely': {
        'description': ('Subresource Integrity (SRI) is not implemented, and scripts '
                        'from a similar origin are not loaded over https'),
```